### ISSUE TITLE
doesn't work on ts and tsx files

### ISSUE THREAD

**Issue by JustFly1984** [zertosh/eslint-plugin-dependencies#19]
I got this warnings on 100% working codebase after porting it to typescript.

```
   2:30  warning  "./injectscript" does not exist           dependencies/no-unresolved
   3:27  warning  "./isbrowser" does not exist              dependencies/no-unresolved
```
